k8s/demo/common/ccd/data-store-api.yaml

```diff
@@ -34,7 +34,6 @@ spec:
         DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-demo
         DATA_STORE_CROSS_JURISDICTIONAL_ROLES: caseworker-caa,caseworker-approver
         CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-demo.service.core-compute-demo.internal
-        DUMMY_RESTART_VAR: 1
```

Comment:
Better to just change the value instead, so easier for people in future wanting to restart?